.github/workflows/cut-rc.yml

```diff
@@ -0,0 +1,598 @@
+name: Cut RC
+on:
+  milestone:
+    types:
+      - closed
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Full RC number (A.B.C-rc.D)
+        required: true
+        default: 0.0.0-rc.1
+        type: string
+      dry-run:
+        description: Dry-run
+        required: false
+        default: true
+        type: boolean
+
+env:
+  main_branch: master
+  docs_repository: openshift/openshift-docs
+  jira_projects: ROX, RS, RTOOLS
+  slack_channel: C03KSV3N6N8 #test-release-automation
```

Comment:
I think we already have this in CircleCI config in plain text. Is it possible to put this in a secret?

Reply:
It is possible. Benefit: single source for all workflows. Disadvantage: only admins have access. As the ID is not necessarily a sensitive information, we might want to keep during testing phase.